packages/store/tests/selector.spec.ts

```diff
@@ -817,4 +799,55 @@ describe('Selector', () => {
       }
     });
   });
+
+  describe('Select errors', () => {
+    @State<number[]>({
+      name: 'tasks',
+      defaults: [1, 2, 3, 4]
+    })
+    class NumberListState {
+      @Selector()
+      static reverse(state: number[]): number[] {
+        return state.reverse();
+      }
+    }
+
+    it('should be correct catch errors from observable with select', fakeAsync(() => {
```

Comment:
Do you need `fakeAsync` here?

Reply:
Yes it's need for test

Reply:
You don't have any asynchronous code in that test

Reply:
If you remove fakeAsync then the code does not work

Reply:
So it's needed to understand why it doesn't work, not putting `fakeAsync` here :relaxed:

Reply:
done